src/Client/ConnectionPool.cpp

```diff
@@ -18,7 +18,7 @@ ConnectionPoolPtr ConnectionPoolFactory::get(
     String client_name,
     Protocol::Compression compression,
     Protocol::Secure secure,
-    Int64 priority)
+    Priority priority)
```

Comment:
but does the comment from `Common/Priority.h` apply in all these places "lower value means higher priority"? I am not sure about that, should we remove the comment then?

Reply:
Yes, the idea is to use the same logic everywhere, am I wrong this is the case for priority here?

Reply:
Indeed looks so, just asked because I did not know if all of them follow the same priority rule